src/latency.c

```diff
@@ -477,18 +474,18 @@ sds createLatencyReport(void) {
 
         if (advise_mass_eviction) {
             report = sdscat(report, "- Sudden changes to the 'maxmemory' setting via 'CONFIG SET', or allocation of "
-                                    "large objects via sets or sorted sets intersections, STORE option of SORT, Redis "
+                                    "large objects via sets or sorted sets intersections, STORE option of SORT, Valkey "
                                     "Cluster large keys migrations (RESTORE command), may create sudden memory "
                                     "pressure forcing the server to block trying to evict keys. \n");
         }
 
         if (advise_disable_thp) {
             report = sdscat(report, "- I detected a non zero amount of anonymous huge pages used by your process. This "
                                     "creates very serious latency events in different conditions, especially when "
-                                    "Redis is persisting on disk. To disable THP support use the command 'echo never > "
+                                    "Valkey is persisting on disk. To disable THP support use the command 'echo never > "
                                     "/sys/kernel/mm/transparent_hugepage/enabled', make sure to also add it into "
```

Comment:
This line is too long (121 characters) so clang-format wants to change the formatting. It's better to fix it like this IMO:

```suggestion
                                    "Valkey is persisting on disk. To disable THP support use the command 'echo never "
                                    "> /sys/kernel/mm/transparent_hugepage/enabled', make sure to also add it into "
```

Then please run `clang-format -i src/latency.c` or fix manually what it reported in the clang-format CI job log.